### PR TITLE
ref(event-entries): Remove 2x attachment fire request

### PR DESCRIPTION
**Pull request by priscilawebdev**
The attachment request (`/projects/${this.props.orgId}/${this.props.projectId}/events/${event.id}/attachments/`) was being fired twice, this PR fixes it and:

- Converts EventEntries from class to func and uses hooks for the state
- Converts test `eventAttachments` from jsx to typescript
- Converts test `eventAttachments` from enzyme to React testing library

### CODE REVIEW

File: static/app/components/events/eventEntries.tsx [hunk 1]
```diff
@@ -70,276 +65,296 @@ type Props = {
...
 
-  componentDidMount() {
-    this.checkProGuardError();
-    this.recordIssueError();
-  }
+      const errors = event.errors;
+      const errorTypes = errors.map(errorEntries => errorEntries.type);
+      const errorMessages = errors.map(errorEntries => errorEntries.message);
 
-  shouldComponentUpdate(nextProps: Props, nextState: State) {
```
Comment (dashed) [inline static/app/components/events/eventEntries.tsx:94]: Would some of these conditional checks need to be moved to `React.memo()`'s 2nd argument?

Comment (priscilawebdev) [reply static/app/components/events/eventEntries.tsx:94]: I did some checks and that wasn't needed

Comment (priscilawebdev) [reply static/app/components/events/eventEntries.tsx:94]: I'll double-check everything on staging  😉

**Review by matejminar — APPROVED** [review]
Looks good to me.

**Pull request merged**